lib/screens/v2/candidate_generator/dup/departures.ex

```diff
@@ -516,6 +513,28 @@ defmodule Screens.V2.CandidateGenerator.Dup.Departures do
     nil
   end
 
+  # If now is after today's last schedule and there are no schedules tomorrow,
+  # we still want a departure row without a time (will show a moon icon)
+  defp get_overnight_departure_for_route(
+         first_schedule_today,
+         last_schedule_today,
```

Comment:
I think this would be a separate issue to fix, but do we need to handle the case when there are no schedules _today_?

The remaining scenarios I can think of that (afaik) aren't already covered are
- No schedules today, no schedules tomorrow
- No schedules today, yes schedules tomorrow

Reply:
I honestly hadn't thought of that. I had always considered no schedules today meaning there was some kind of issue, but this task is specifically for routes that don't run on certain days. Because predictions take priority, I think it's safe to assume a route not running today is in overnight mode. Throughout the day, predictions would show as long as there are any, and we would always have an overnight row in the queue when we start running out of predictions. Does that sound right?

Reply:
Had a chat with PaulH. If a route has no schedules today, don't assume it's in overnight mode. This does mean that a route will not show overnight mode at all on the service day it is not running, but that's ok. We will let it have the opportunity to show again when it reaches the next service day.